src/ProjectOrigin.WalletSystem.IntegrationTests/MigrationTests.cs

```diff
@@ -0,0 +1,44 @@
+using Dapper;
+using FluentAssertions;
+using ProjectOrigin.WalletSystem.Server.Database;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ProjectOrigin.WalletSystem.IntegrationTests;
+
+public class MigrationTest : IClassFixture<PostgresDatabaseFixture>
```

Comment:
I thought that this was deleted...
This is indirectly tested in the other integration tests

Reply:
I think it should be rewritten, so we can test that we can start the server with a parameter to only do the migration and stop, then we know it would work as an init container.

Reply:
That makes sense, but I see the task of rewriting this test to be a part of creating the Helm Chart issue and not this issue. Do you agree?

Reply:
yes